docs/api.md

```diff
@@ -1,1880 +1,2210 @@
 
-___Headers___
+  <!--begin-response-body-get-accountsessions-deviceName-->
+  
+  <!--end-response-body-get-accountsessions-deviceName-->
 
-The request must include a Hawk header that authenticates the request (including payload) using a `sessionToken` received from `/v1/session/create`.
+* `deviceType`: *string, max(16), required, allow(null)*
 
-```sh
-curl -v \
--X POST \
--H "Host: api-accounts.dev.lcip.org" \
--H "Content-Type: application/json" \
--H 'Authorization: Hawk id="d4c5b1e3f5791ef83896c27519979b93a45e6d0da34c7509c5632ac35b28b48d", ts="1373391043", nonce="ohQjqb", hash="vBODPWhDhiRWM4tmI9qp+np+3aoqEFzdGuGk0h7bh9w=", mac="LAnpP3P2PXelC6hUoUaHP72nCqY5Iibaa3eeiGBqIIU="' \
-https://api-accounts.dev.lcip.org/v1/recovery_email/resend_code
-```
+  <!--begin-response-body-get-accountsessions-deviceType-->
+  
+  <!--end-response-body-get-accountsessions-deviceType-->
 
-### Response
+* `deviceCallbackURL`: *string, uri({ scheme: 'https' }), max(255), optional, allow(''), allow(null)*
 
-Successful requests will produce a "200 OK" response with an empty JSON body:
+  <!--begin-response-body-get-accountsessions-deviceCallbackURL-->
+  
+  <!--end-response-body-get-accountsessions-deviceCallbackURL-->
 
-```json
-{}
-```
+* `deviceCallbackPublicKey`: *string, max(88), regex(URLSAFEBASE64), optional, allow(''), allow(null)*
 
-Failing requests may be due to the following errors:
+  <!--begin-response-body-get-accountsessions-deviceCallbackPublicKey-->
+  
+  <!--end-response-body-get-accountsessions-deviceCallbackPublicKey-->
 
-* status code 400, errno 106:  request body was not valid json
-* status code 401, errno 109:  invalid request signature
-* status code 401, errno 110:  invalid authentication token
-* status code 401, errno 111:  invalid authentication timestamp
-* status code 411, errno 112:  content-length header was not provided
-* status code 413, errno 113:  request body too large
-* status code 401, errno 115:  invalid authentication nonce
+* `deviceCallbackAuthKey`: *string, max(24), regex(URLSAFEBASE64), optional, allow(''), allow(null)*
 
+  <!--begin-response-body-get-accountsessions-deviceCallbackAuthKey-->
+  
+  <!--end-response-body-get-accountsessions-deviceCallbackAuthKey-->
 
-## POST /v1/recovery_email/verify_code
+* `isDevice`: *boolean, required*
 
-Not HAWK-authenticated.
+  <!--begin-response-body-get-accountsessions-isDevice-->
+  
+  <!--end-response-body-get-accountsessions-isDevice-->
 
-This is an endpoint that is used to verify tokens and additional emails for an account. If a valid token code is detected, the account email and tokens will be set to verified. If a valid email code is detected, the email will be marked as verified.
+* `isCurrentDevice`: *boolean, required*
 
-The verification code will be a random token, delivered in the fragment portion of a URL sent to the user's email address. The URL will lead to a page that extracts the code from the 
```

Comment:
can we put a short warning in the beginning of the doc. just after the header:

```
> This document is automatically generated.
```
With a link to the paragraph below.